weldx/asdf/schemas/weldx.bam.de/weldx/datamodels/single_pass_weld-1.0.0.schema.yaml

```diff
@@ -17,52 +26,406 @@ properties:
         $ref: "http://weldx.bam.de/schemas/weldx/process/GMAW-1.0.0"
       shielding_gas:
         tag: "tag:weldx.bam.de:weldx/aws/process/shielding_gas_for_procedure-1.0.0"
-  equipment:
-    type: array
-    items:
-      tag: "tag:weldx.bam.de:weldx/equipment/generic_equipment-1.0.0"
-  measurements:
-    type: array
-    items:
-      tag: "tag:weldx.bam.de:weldx/measurement/measurement-1.0.0"
+      weld_speed:
+        description: |
+          The constant weld speed of the welding TCP movement.
+        tag: "tag:stsci.edu:asdf/unit/quantity-1.1.0"
+        wx_unit: "m/s"
+        shape: [1]
```

Comment:
why isn't this a scalar?

Reply:
it is (or should be at least) as indicated by `shape: [1]`

Reply:
I see. Explicit is better then implicit (zen of Python).